dev/tasks/tasks.yml

```diff
@@ -29,6 +29,7 @@ groups:
 
   wheel:
     - wheel-*
+    - python-sdist
```

Comment:
I've added the `sdist` as part of the wheels job as several times we want to also trigger it when we trigger the wheels group but happy to remove it from here if we want to have it separate